crates/interpreter/src/instructions/opcode.rs

```diff
@@ -463,13 +470,8 @@ impl OpCode {
     }
 
     /// Instantiate a new opcode from a u8 without checking if it is valid.
-    ///
-    /// # Safety
-    ///
-    /// All code using `Opcode` values assume that they are valid opcodes, so providing an invalid
-    /// opcode may cause undefined behavior.
     #[inline]
-    pub unsafe fn new_unchecked(opcode: u8) -> Self {
+    pub fn new_unchecked(opcode: u8) -> Self {
```

Comment:
Made it unsafe as OpCode should not be possible to have an undefined byte

Reply:
It's not memory unsafe and I'd like to use this to format any opcode, like `("{}",  Opcode::new_unchecked(op_byte))`

Reply:
It is better to not have undefined byte in struct